1.1/learn/how-to-document-ballerina-code.md

```diff
@@ -64,6 +64,19 @@ The supported structure of documentation syntax is as follows:
 #            <return_parameter_description_line_2>
 ```
 
+>**Tip:** As a best practice, do not add a full stop at the end of the first sentence of the parameter and return type descriptions. Instead, add a fullstop only at the end of a function description. For Example,
```

Comment:
IMHO, this description is bit misleading. It says `do not add a full stop at the end of the first sentence of the parameter and return type descriptions.` 

But actually we should add it to the first sentence if there are two sentences. We need to omit if only if we have only one sentence.

Reply:
+1. @praneesha please add this.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1269.